javascript/javascript2/week1/nameGenerator.js

```diff
@@ -0,0 +1,62 @@
+
+
+const spiritAnimal=["The fullmoon wolf","merciful lion","fast rabit","greedy Dog"];
+
+document.getElementById("myBtn").addEventListener("click", function() {
+    let userInput=document.getElementById("myText").value;
+
+    if(userInput=="tariq")
+    {
+        let name=spiritAnimal[Math.floor(Math.random() * spiritAnimal.length)];
+        let res=userInput+":"+name;
+        document.getElementById('para').textContent=res; 
+        
+    }  
+    
+    if(userInput=="")
+    {
+        let y="please enter here"
+        document.getElementById("myText").value=y;
+    }
+
+});
+
+function empty()
+{
+    document.getElementById("myText").value="";
+    
+    document.getElementById('para').textContent="";
+}
+
+// code for mouseover
+
+/*
+const spiritAnimal=["The fullmoon wolf","merciful lion","fast rabit","greedy Dog"];
+
+document.getElementById("myBtn").addEventListener("mouseover", function() {
+    let userInput=document.getElementById("myText").value;
+
+    if(userInput=="tariq")
+    {
+        let name=spiritAnimal[Math.floor(Math.random() * spiritAnimal.length)];
+        let res=userInput+":"+name;
+        document.getElementById('para').textContent=res; 
+        
+    }  
+    
+    if(userInput=="")
+    {
+        let y="please enter here"
+        document.getElementById("myText").value=y;
+    }
+
+});
+
+function empty()
+{
+    document.getElementById("myText").value="";
+    
+    document.getElementById('para').textContent="";
+}
+*/
+
```

Comment:
Your code dose the job. however, only if the name "tariq" is put in by the user. Why is that? Further, the `mouseover` example is commented out. couldnt you create a dropdown list, where I can choose the `addEvenListener` behaviour?